From 58a06f33313ad23d406514a03ab8d8f13e3cc44e Mon Sep 17 00:00:00 2001
From: Karl Palsson <karlp@etactica.com>
Date: Fri, 4 Dec 2020 21:33:18 +0000
Subject: [PATCH] apps: mosquitto_ctrl: don't hard set paths

In a cross compile environment, these paths will be added elsewhere, and
in a local environment, they should already be covered by the system
compiler.  Simply drop them.

Signed-off-by: Karl Palsson <karlp@etactica.com>
Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 apps/mosquitto_ctrl/Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/apps/mosquitto_ctrl/Makefile b/apps/mosquitto_ctrl/Makefile
index 59b23596..3a4843bf 100644
--- a/apps/mosquitto_ctrl/Makefile
+++ b/apps/mosquitto_ctrl/Makefile
@@ -8,8 +8,7 @@ else
 LIBMOSQ:=../../lib/libmosquitto.a
 endif
 
-LOCAL_CPPFLAGS:=-I/usr/include/cjson -I/usr/local/include/cjson -I../mosquitto_passwd
-LOCAL_LDFLAGS:=-L/usr/local/lib
+LOCAL_CPPFLAGS:=-I../mosquitto_passwd
 
 OBJS=	mosquitto_ctrl.o \
 		client.o \
-- 
2.20.1

